### PR TITLE
Dm 1644

### PR DESCRIPTION
**Pull request by JoshingYou1**
### JIRA issue link
https://agile6.atlassian.net/browse/DM-1644

## Description - what does this code do?
UX tweaks and fixes an issue regarding the checkbox options for practice partners and practice departments.

## Testing done - how did you test it/steps on how can another person can test it 
**Doesn't require IE:**
1. Log in as an admin/approver.
2. Browse to both the overview page and complexity page of the practice editor.
3. Check or uncheck partners or departments as desired.
4. Save your progress.
5. Ensure the changes are appropriately reflected.
6. Make sure UX changes marked in green(that don't specify IE) in the header and practice page tabs in the linked spreadsheet are correct.

**Requires IE:**
1. Make sure UX changes marked in green(that specify IE) in the header and practice page tabs in the linked spreadsheet are correct.

## Link to mock-ups/mock ups (image file if you have it) (if applicable)
https://www.figma.com/file/DDUSxYiD1MaWNrGFmMjY8d8a/Diffusion-Marketplace-Designs?node-id=1745%3A27776

## Acceptance criteria
- [Follow this spreadsheet: https://docs.google.com/spreadsheets/d/1p3Sc07bkglWnDEuEyHE5q4MeQ7Th3_3D6xNv8W_9PPE/edit#gid=466692682]
- [These need to be vetted first to make sure they’re still issues]
- [Work from spreadsheet through different tabs on all items not already fixed or made redundant]

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [X] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/views/practices/complexity.html.erb [hunk 1]
```diff
@@ -41,13 +41,13 @@
...
-                            <% Department.order("CASE WHEN name = 'None' THEN 1 WHEN name = 'All' THEN 2 ELSE 3 END, name").each do |department| %>
+                            <% Department.order("CASE WHEN name = 'None' THEN 1 WHEN name = 'All departments equally - not a search differentiator' THEN 2 ELSE 3 END, name").each do |department| %>
                               <% if department.name == 'None' %>
                                 <div class='usa-checkbox grid-col-6'>
                                   <input type='checkbox' class='usa-checkbox__input no-department-input' id="practice_department_<%= @no_departments.id %>" name="practice[department][<%= @no_departments.id %>][value]" <%= !@practice.departments.find_by(name: 'None').nil? ? "checked= 'true'" : "" %>>
                                   <label class='usa-checkbox__label' for="practice_department_<%= @no_departments.id %>">None</label>
                                 </div>
-                              <% elsif department.name == 'All' %>
+                              <% elsif department.name == 'All departments equally - not a search differentiator' %>
```
Comment (ahay-agile6) [inline app/views/practices/complexity.html.erb:50]: wondering if we should do a unique short name or something here

Comment (ahay-agile6) [reply app/views/practices/complexity.html.erb:50]: gonna merge this for now.
i'll think about this more when it bites us again :smile:

Comment (JoshingYou1) [reply app/views/practices/complexity.html.erb:50]: Can I go in and write a task to assign a short name to this department?

Comment (ahay-agile6) [reply app/views/practices/complexity.html.erb:50]: i think i also want `short_name` to be unique

Comment (ahay-agile6) [reply app/views/practices/complexity.html.erb:50]: we decided to wait on this :)

**Comment by emyl3** [comment]
Not sure if this in scope of this PR... but when I checked the departments under Complexity and saved
Screen Shot 2020-04-28 at 4 09 10 PM
I expected the number of departments under the Complexity section to change as well:
Screen Shot 2020-04-28 at 4 05 56 PM

I checked the DB and the `DepartmentPractice` was correctly updated...

Looks like that view is being driven here `/app/helpers/practices_helper.rb` in `line 23` not sure where `practice.number_departments` is being set though... Once again, not sure if this is in scope but was curious about this 😅

**Pull request merged**